packages/theme/components/FormComponents/EmailInput.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <SfInput
+      @click="e => $emit('click', e)"
+      @input="e => $emit('input', e)"
+      v-bind="{...$attrs, ...$props}"
+      :class="{'disable-input': disabled}"
+      :valid="!errors[0]"
+      :errorMessage="errors[0]"
+    />
+  </div>
+</template>
+
+<script>
+import {
+  SfInput
+} from '@storefront-ui/vue';
+
+export default {
+  name: 'EmailInput',
+  props: {
+    disabled: {
+      type: Boolean,
+      default() {
+        return false;
+      }
+    },
+    label: {
+      type: String,
+      default() {
+        return '';
+      }
+    },
+    name: {
+      type: String,
+      default() {
+        return 'email';
+      }
+    },
+    value: {
+      type: String
```

Comment:
To increase the readability, use either the `default` or `required` property when declaring props. While we're using the `SfInput`, it defaults to "". It should be removed from props anyways because we should left specifying props (we don't use directly) to the user.

Reply:
Got it!